streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/StickyTaskAssignorTest.java

```diff
@@ -64,6 +74,28 @@ public void shouldAssignOneActiveTaskToEachProcessWhenTaskCountSameAsProcessCoun
         }
     }
 
+    @Test
+    public void shouldAssignTopicGroupIdEvenlyAcrossClientsWithNoStandByTasks() {
+        createClient(p1, 2);
+        createClient(p2, 2);
+        createClient(p3, 2);
+
+        final StickyTaskAssignor taskAssignor = createTaskAssignor(task10, task11, task22, task20, task21, task12);
```

Comment:
Should we randomly shuffle the input list before calling `createTaskAssignor(...)` ?

Reply:
`createTaskAssignor` shuffles the list internally